apps/dh/e2e-dh/src/integrations/app-shell.spec.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { test, expect } from '@playwright/test';
+
+import * as appShell from '../support/app-shell.po';
+
+test.describe('Application shell', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/metering-point/search');
+  });
+
+  test('the application title is displayed', async ({ page }) => {
+    test.slow();
+    await page.waitForTimeout(5000);
+    console.log('-------------------------------------');
+    console.log('the application title is displayed');
+    console.log(await page.innerHTML('dh-metering-point-search'));
+    console.log('-------------------------------------');
+    await expect(appShell.getTitle(page)).toHaveText('Målepunkter');
```

Comment:
issue: The Danish translation file was updated recently (#230) and the title above the search input is now "Søg målepunkter". So something to be aware of. The changes are not on `main`.